### DISCUSSION TITLE
Eufy Lock

### DISCUSSION THREAD

**Discussion by johnrobledo** [fuatakgun/eufy_security#67] [Ideas]
Would there be any way to add support for the Eufy wifi lock?

**Comment by fuatakgun** [comment]
i am interested to make it happen but I need to dump some data from an existing user. I believe locks are not shareable as if they were cameras, so if an owner commits to it, it will be a long journey for both of us :)

**Comment by johnrobledo** [comment]
Of course! I would be willing to work with you. Also, I believe they are shareable with other eufy accounts.

**Reply by fuatakgun** [reply]
Are you using it in day to day life? i might need to play with it and it looks not safe to play with it while holding your home secure :-)

**Reply by johnrobledo** [reply]
Its alright...the door has another lock and we live in a gated neighborhood.

**Reply by fuatakgun** [reply]
Ok, give me a couple of days, i will create the initial classes and let you know for further steps.

**Reply by johnrobledo** [reply]
Thanks!

**Reply by johnrobledo** [reply]
Hi...just a question...were you the same person who replied to me on Mark Watt's youtube video? I didn't even notice that when I asked the question.

**Reply by fuatakgun** [reply]
Yes it is me, Fuat Akgün as seen on YouTube.

**Reply by fuatakgun** [reply]
I thought it was clear that i am the developer of integration and asking you to create an issue/discussion about this. Happy to see that we have triggered this

**Reply by johnrobledo** [reply]
Yes! If you were trying to let me know that, I didn't realize! Again, thanks so much for doing this.

**Reply by johnrobledo** [reply]
Hi! Just wanted to see how's it going! No rush...just wanted to check

**Comment by fuatakgun** [comment]
I know what i need to do, but weekend seems to be the most suitable time. In the meantime, you can setup everything in your ha instance, i will update integration directly as there will be no impact to camera users, this will be introduction for lock itself.

**Reply by johnrobledo** [reply]
Hi...just wondering how it's going!

**Comment by fuatakgun** [comment]
i have setup my local test environment, please share one of your, your lock with me following readme so I can debug locally.

**Reply by johnrobledo** [reply]
I share it to the email on your account correct?

**Reply by johnrobledo** [reply]
If so, i have shared it.

**Reply by fuatakgun** [reply]
Hey, can you share again? I see the invitation as an email but it is not visible inside the app.

If you are based in US, please try with this eufydeveloper@gmail.com

**Comment by johnrobledo** [comment]
Okay I did it

**Reply by fuatakgun** [reply]
it works, get latest codebase and enjoy.

you can cancel the sharing. if you have any issues, create an issue.

**Comment by johnrobledo** [comment]
Nice! Just to make sure the integration for eufy security correct?

**Reply by fuatakgun** [reply]
please follow readme over here : https://github.com/fuatakgun/eufy_security :)
